core/tests/bank_forks.rs

```diff
@@ -302,11 +340,12 @@ mod tests {
         // Purge all the outdated snapshots, including the ones needed to generate the package
         // currently sitting in the channel
         bank_forks.purge_old_snapshots();
-        let mut snapshot_paths = snapshot_utils::get_snapshot_paths(&snapshots_dir);
-        snapshot_paths.sort();
```

Comment:
Nice. Yeah, we don't need to `.sort()` indeed. It's done inside `get_snapshot_paths()` already.